style.css

```diff
@@ -371,18 +383,293 @@ header {
 }
 
 #comment {
-  width: 100%;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
   height: 150px;
-  margin-bottom: 20px;
   padding: 10px 16px;
-  font-size: 17px;
-  line-height: 20px;
-  font-weight: 400;
   font-family: 'Poppins', sans-serif;
 }
 
-#form > input {
-  font-family: 'Poppins', sans-serif;
+@media screen and (max-width: 767px) {
+  .main-menu {
+    display: none;
+  }
+}
```

Comment:
[OPTIONAL] Kindly, using media-queries for this section is not necessary, because everything above the
@media screen and (max-width: 767px) {
It is considered mobile